exchanges/kraken/kraken.go

```diff
@@ -119,15 +134,26 @@ func (k *Kraken) GetAssets() (map[string]*Asset, error) {
 }
 
 // GetAssetPairs returns a full asset pair list
-func (k *Kraken) GetAssetPairs() (map[string]*AssetPairs, error) {
+// Parameter 'info' only supports 3 strings: "fees", "leverage", "margin"
+func (k *Kraken) GetAssetPairs(assetPairs []string, info string) (map[string]AssetPairs, error) {
```

Comment:
This function was untested before, but please add one now, also testing with these parameters